### PR TITLE
Implementation of last and last_or operators.

### PR DESCRIPTION
**Pull request by achary**
This a `last` and `last_or` operator implementation and unit tests.

### CODE REVIEW

File: src/ops/last.rs [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+  fn to_shared(self) -> Self::Shared {
+    SharedOp(LastOrOp {
+      source: self.source.to_shared(),
+      default: self.default,
+      last: self.last,
+    })
+  }
+}
+
+pub struct LastOrSubscribe<S, T> {
```
Comment (M-Adoo) [inline src/ops/last.rs]: LastOrSubscribe -> LastOrObserver

`LastOrSubscribe` is a better name, xxxSubscribe is a 0.1 style name 😂.

Comment (achary) [reply src/ops/last.rs]: Thanks. yes. Done.

**Comment by M-Adoo** [comment]
please resolve the conflicts

**Comment by achary** [comment]
Here is the pain point. CHANGELOG.md becomes a source of never-ending merge conflicts. Continuous changes to it on rebase/merge from master, keep re-triggering flaky smoke unit tests tests that fail occasionally.

I would recommend:
- prioritise up fixing the unit test issue. If they are not reliable, maybe should be switched off for now? They are not trustworthy anyway.
- replace manual change log editing with one of automated changelog generation (I think Gitlab has a solution addressing this, there is also `gitchangelog` tool that could automate it etc).

**Comment by M-Adoo** [comment]
- sorry for the unit test not reliable, i'll try to fix it this week, before it be fixed you need to manual re-triggger it.
- Many tools can generate it, but i think write a change log it's a friendly way for user, we also need to track an **Unreleased** section.  But to resolve "never-ending merge conflicts", let not force to  require a change log update in pr from now on. We regularly update change log separately, of course must always update  it before release a version.

**Pull request merged**